source/hardware/controllers/hercules_djcontrol_mp3_e2.rst

```diff
@@ -3,589 +3,123 @@ Hercules DJControl MP3 e2 / MP3 LE / Glow
-| corresponding button  |                       |                       |
-| to obtain             |                       |                       |
-| supershifted          |                       |                       |
-| behavior.             |                       |                       |
-+-----------------------+-----------------------+-----------------------+
-| Switch deck A (left)  | Supershift + Load A   | 18                    |
-| between Channel1 and  |                       |                       |
-| Channel3 (also apply  |                       |                       |
-| to Sampler1 and       |                       |                       |
-| Sampler3)             |                       |                       |
-+-----------------------+-----------------------+-----------------------+
-| Switch deck B (right) | Supershift + Load B   | 18                    |
-| between Channel2 and  |                       |                       |
-| Channel4 (also apply  |                       |                       |
-| to Sampler2 and       |                       |                       |
-| Sampler4)             |                       |                       |
-+-----------------------+-----------------------+-----------------------+
-
-Library
-^^^^^^^
-
-.. figure:: ../../_static/controllers/hercules_djcontrol_mp3_e2_library.jpg
-   :align: center
-   :width: 100%
-   :figwidth: 100%
-   :alt: Hercules DJControl MP3 e2 (library)
-   :figclass: pretty-figures
-
-   Hercules DJControl MP3 e2 (library)
-
-
-=========================================================================== ============================= ======
-Function                                                                    Control                       number
-=========================================================================== ============================= ======
-Select prev/next playlist                                                   Supershift + Deck A Jog Wheel 16
-Toggles (expands/collapses) the currently selected library sidebar item     Folder                        6
-Go one track down                                                           Up arrow                      8
-Go one track up                                                             Shift + Up arrow              8
-Select prev/next track                                                      Supershift + Deck B Jog Wheel 16
-Loads the currently highlighted track into the corresponding deck (A or B)  Load A/B                      18
-Loads the current highlighted track into the corresponding sampler (1 or 2) Shift + Load A/B              18
-=========================================================================== ============================= ======
-
-Master/Headphones/Microphone
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. figure:: ../../_static/controllers/hercules_djcontrol_mp3_e2_masterheadmicro.jpg
+.. figure:: ../../_static/controllers/hercules_djcontrol_mp3_e2_mapping.png
```

Comment:
Do you habe an SVG file for this? That would be even better than a JPG.

Reply:
Sorry but I don't have an SVG file for this.

Reply:
Where did you get the image from. Sometimes it 's possible to open the owner manual PDF with Inkscape.

Reply:
The image was added by the last contributor before me. I think I've succeed to add an SVG file for it. Learned something new, that's cool :)